lumigator/python/mzai/backend/backend/services/jobs.py

```diff
@@ -99,13 +100,18 @@ def create_inference_job(self, request: JobCreate) -> JobResponse:
             "dataset_path": dataset_s3_path,
             "max_samples": request.max_samples,
             "storage_path": storage_path,
+            "output_field": request.output_field,
             "model_url": model_url,
             "system_prompt": request.system_prompt,
+            "max_tokens": request.max_tokens,
+            "frequency_penalty": request.frequency_penalty,
+            "temperature": request.temperature,
+            "top_p": request.top_p,
         }
 
         # load a config template and fill it up with config_params
-        if request.config_infer_template is not None:
-            config_template = request.config_infer_template
+        if request.config_template is not None:
```

Comment:
@veekaybee in this case the config template is just an input parameter and I thought about keeping the name identical, so at some point this code could be factored out as being practically identical across different jobs (also, we might just remove it if we end up using pydantic configs here too...)

Reply:
yep so you created the same parameter but just two different jobs, makes sense